update-travis-config.sh

```diff
@@ -27,6 +27,7 @@ print_static_configuration() {
 	  - ./update-docker-files.sh
 	  - ./install-img.sh
 	  - ./install-manifest-tool.sh
+      - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh && . ./install-jdk.sh -F 11
```

Comment:
It's using a mix of tabs and spaces. The tab is removed from the generated content:

```suggestion
	  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh && . ./install-jdk.sh -F 11
```